### ISSUE TITLE
Unable to download file from...

### ISSUE THREAD

**Issue by cxminer** [dlmanning/gulp-sass#349]
When running:
> npm install gulp-sass

I get:
> Can not download file from https://raw.githubusercontent.com/sass/node-sass-binaries/v2.1.1/win32-x64-node-3.3/binding.node

After that I see there is a fallback? That tries to build from source? But fails because of:
> Error: Can't find Python executable "python", you can set the PYTHON env variable.

Do I need to install python in order to use this package?

Output of `npm -v` is `2.14.3`.
My OS is `Windows 8 x64`.
I tried installing with iojs and nodejs, result was the same.

Full output can be found at [this Gist](https://gist.github.com/cxminer/63c0393d7976c27c3685#file-gistfile1-txt).

**Comment by cxminer** [comment]
I just found out more about this. This error occurs when I try to run `ionic setup sass`. I found out that inside `package.json` of this module (`gulp-sass`) there is a dependency called `node-sass` and the version used is `^2.0.1` which at the moment translates to `2.1.1`, this version seems to be bugged and doesn't download the necessary `libsass`. If we were to change the version to `^3.0.0` it would successfully install. So basically this is not a problem with `gulp-sass` I believe. My bad...
